### PR TITLE
Develop pull request

### PR DESCRIPTION
**Pull request by dmin0211**

### CODE REVIEW

**Comment by hyunmindev** [comment]
test

**Review by hyunmindev — COMMENTED** [review]
comment test

File: src/components/community/CommunityCardTitle.vue [hunk 1]
```diff
@@ -21,11 +21,9 @@ export default {
       },
     },
   },
-  data() {
-    return {
-      currentTypeString: '',
-    };
-  },
+  data: () => ({
```
Comment (hyunmindev) [inline src/components/community/CommunityCardTitle.vue:24]: nice arrow data functions!

File: src/views/community/question/QuestionPosts.vue [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<template>
```
Comment (hyunmindev) [inline src/views/community/question/QuestionPosts.vue:1]: this file is nice

**Review by hyunmindev — APPROVED** [review]
LGTM

File: src/router/routes.js [hunk 3]
```diff
@@ -113,9 +114,14 @@ export default [
...
         path: 'question',
-        redirect: { name: 'question thread' },
+        redirect: { name: 'question posts' },
         component: CommunityQuestion,
         children: [
+          {
+            name: 'question posts',
+            path: '',
+            component: QuestionPosts,
+          },
```
Comment (dmin0211) [inline src/router/routes.js:124]: terms change

**Pull request merged**